src/pages/scripts/concepts/script-lifespan/index.md

```diff
@@ -0,0 +1,65 @@
+# Script lifespan
```

Comment:
@kareemmyk  and @anoopvalomkot  Please review this tutorial to see if the description is acurate.

Reply:
@padmkris123 

Looks correct, but I feel it isn't completely describing the await aspect, unless we have it covered elsewhere. 
There are some sample code under https://wiki.corp.adobe.com/pages/viewpage.action?spaceKey=UXP&title=UXP+Scripting%3A+Script+Developer+Guide

Including one of them might give bring good clarity. Please check once.

Reply:
Which one in particular are you referring to @anoopvalomkot?  Keep in mind that this section is aiming to explain the 'concept' first.  Its not meant to showcase a full-fledged API/recipe. We havent yet spoken about any of the APIs at this point.

Reply:
ok, do we have any other section with sample programs on async script?  In the sample code within the link I shared, you will notice that it is not as simple as calling await. Sometimes user has to implement APIs(which register handlers for async events) that return promise and call await on them during API execution.

Reply:
As discussed, let's change the example to 
```js
console.log("Start of execution.");

function foo() {
    console.log("Greetings from Foo: Hello");
}

foo();

console.log("End of execution.");
```